src/specs/api.yaml

```diff
@@ -562,6 +562,9 @@ components:
         maxExpression:
           type: number
           description: The maximum expression value of that gene
+        expressionType:
```

Comment:
can you add a default in here similarly to how it is in the response, just to have it consistent?

Reply:
Default response added in new commit 8368025dcb00405a64939ba13ce8f9106c045166